src/main/java/org/terracotta/statistics/extended/SamplingSupport.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics.extended;
+
+/**
+ * @author Ludovic Orban
+ */
+public interface SamplingSupport {
+
+  boolean expire(long expiry);
+
+  void setAlwaysOn(boolean enable);
```

Comment:
`setAlwaysOn ` is the method that would avoid a stats to stop being computed, used for testing purposes, right ?